sdk/lib/atl/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+
+add_library(atl_classes INTERFACE)
+
+target_include_directories(atl_classes INTERFACE ${CMAKE_CURRENT_SOURCE_DIR})
+
+target_compile_definitions(atl_classes INTERFACE
+    "$<$<AND:$<COMPILE_LANGUAGE:CXX>,$<NOT:$<BOOL:$<TARGET_PROPERTY:WITH_CXX_EXCEPTIONS>>>>:_ATL_NO_EXCEPTIONS>")
```

Comment:
Does this use the property WITH_CXX_EXCEPTIONS from the actual project that `atl_classes` is linked into, or from `atl_classes` ?

Reply:
This is transitive, see here:
https://cmake.org/cmake/help/v3.17/manual/cmake-generator-expressions.7.html#id14